### PR TITLE
NETOBSERV-1266 Netflow traffic tab crash

### PR DESCRIPTION
**Pull request by jpinsonneau**
NetflowTab props is updating a log and retrigger NetflowTraffic page endlessly.
- added a state + check on previous prop values to avoid infinite loop
- ensure `navFunc` is loaded when `navigate` is called

### CODE REVIEW

**Comment by Amoghrd** [comment]
oh didnt see the branch is out of date with master. If you could rebase with master @jpinsonneau I can go ahead with testing it if its ready

**Comment by jpinsonneau** [comment]
Fixed testing since enzyme doesn't manage `useEffect` hook https://github.com/netobserv/network-observability-console-plugin/pull/377/commits/5d6589766c4110eaff2345093042051bb9144b0d

Also took the opportunity to add some tab examples to `standalone` mode https://github.com/netobserv/network-observability-console-plugin/pull/377/commits/2bbe32072ee5617c7cc4ac3475b06573d0bd0b77
image
image

**Comment by jpinsonneau** [comment]
I'm generating a new image just in case but there are no changes in the behavior if you already started your testing @Amoghrd :+1:

File: web/src/app.tsx [hunk 1]
```diff
@@ -52,17 +68,41 @@ export class App extends React.Component<{}, AppState> {
   };
 
   private getPageContent = (id: string) => {
+    console.debug('getPageContent', id);
     switch (id) {
-      case 'netflow-traffic-parent':
+      case 'pod-tab':
```
Comment (jotak) [inline web/src/app.tsx:73]: what is the rationale for having here only pod/namespace/node and not all the supported kinds? Is it only used in tests?

Comment (jpinsonneau) [reply web/src/app.tsx:73]: yes it's just for local dev purposes and it's hardcoding "test" as name and "default" as namespace

**Review by jotak — APPROVED** [review]
code LGTM (not tested)

**Pull request merged**